.pre-commit-config.yaml

```diff
@@ -1,18 +1,32 @@
 repos:
+  - repo: local
+    hooks:
+      - id: terraform-providers-lock-staged
+        name: Terraform Providers Lock (on staged .terraform.lock.hcl files)
+        entry: ./.utils/terraform_lock_precommit.sh
+        language: script
+        pass_filenames: false
+        verbose: true
+
   - repo: https://github.com/antonbabenko/pre-commit-terraform
-    rev: v1.83.0
+    rev: v1.96.1
     hooks:
+      - id: terraform_tflint
+        args:
+          - --args=--disable-rule terraform_required_version
+          - --args=--config=__GIT_WORKING_DIR__/.tflint.hcl
       - id: terraform_fmt
       - id: terraform_docs
+        name: terraform_docs on modules
         args:
+          # - --hook-config=--create-file-if-not-exist=true
```

Comment:
Why is this removed?

Reply:
It’s commented out because it wasn’t initially included. Many resources currently lack a `README.md` file, and I wanted to discuss whether we should create them. 
To maintain consistency, I believe we should automate their creation.